src/router/req/authorization.py

```
@@ -52,16 +52,16 @@ def expiration_time():
 
 def gen_refresh_token():
     prefix = str(uuid.uuid4()).replace('-', '')
-    expiration = expiration_time()
+    # 編碼於 token 尾端的過期時間需對齊 cookie max_age (REFRESH_TOKEN_TTL)，
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)